Wiki/Modules/FvwmBacker/index.md

```diff
@@ -3,14 +3,94 @@ title: FvwmBacker
 type: module
 weight: 430
 description: |
-  This module will change the backgound/wallpaper when changing desktops.
+  This module will change the background/wallpaper when changing desktops.
   Any command can be executed when changing desktops so this module can
   be used for more than just switching the background.
 ---
 # FvwmBacker
 
-The FvwmBacker module provides functionality to change the  background when
-changing  desktops.  Any command can be executed to change the backgrounds.
-Actually, any arbitrary command can be sent to fvwm to execute, so  you
+* TOC
+{:toc}
+
+The FvwmBacker module provides functionality to change the background when
+changing  desktops. Any command can be executed to change the backgrounds.
+Actually, any arbitrary command can be sent to fvwm to execute, so you
 could also do things such as changing window border colors, etc.
 
+You can use the FvwmPager to move from screen to screen through each virtual
+desktop set. Now that you can move easily between the desktops you can further
+personalize them by giving them individual background colors.
```

Comment:
This sentence and mentioning FvwmPager here is a bit awkward to me. I do like that you link to it in the example below, and show that you can make the pager and the desktop match. But up here, there is no need to mention the pager, just focus on this can be used to run commands to set backgrounds on desk and/or page changes.